### PR TITLE
Update Readme for NNEF

### PR DESCRIPTION
**Pull request by hansely**

### CODE REVIEW

File: model_compiler/README.md [hunk 1]
```diff
@@ -88,7 +90,7 @@ OPTIONS:
 To convert a NNEF model into AMD NNIR model:
 
 ``` 
-% python nnef_to_nnir.py <nnefInputFolder> <nnirOutputFolder>
```
Comment (kiritigowda) [inline model_compiler/README.md:91]: We have to make sure all the scripts are going to work with python3

**Comment by kiritigowda** [comment]
@hansely and @LakshmiKumar23 any update on this? We need to make sure all the model compiler scripts work with python3. After verification, we need to update all instructions to indicate using python3.

Once, we verify. I will update the scripts and apps to use python3

**Comment by kiritigowda** [comment]
@hansely & @LakshmiKumar23 any updates on this?

**Comment by LakshmiKumar23** [comment]
> @hansely and @LakshmiKumar23 any update on this? We need to make sure all the model compiler scripts work with python3. After verification, we need to update all instructions to indicate using python3.
> 
> Once, we verify. I will update the scripts and apps to use python3

@kiritigowda Haven't looked into this yet. I can start changing a the scripts to be python3 compatible

**Comment by kiritigowda** [comment]
@LakshmiKumar23 is this good to merge with your PRs?

**Comment by LakshmiKumar23** [comment]
> @LakshmiKumar23 is this good to merge with your PRs?

@kiritigowda Yes, this PR changes only the NNEF README. So once #404 is merged, this can be merged too. I am waiting for all the tests to pass so we know what new dependencies needs to be added to README

**Pull request merged**